bindings/core/src/lib.rs

```diff
@@ -49,7 +49,7 @@ pub struct WalletOptions {
 }
 
 impl WalletOptions {
-    pub async fn build_manager(&self) -> iota_sdk::wallet::Result<Wallet> {
+    pub async fn build_manager(self) -> iota_sdk::wallet::Result<Wallet> {
```

Comment:
Just seeing this now but this should have been renamed a while ago. Can you rename to either `build_wallet` or simply `build` please? (+ changelog)